packages/p2p/src/components/advertiser-page/advertiser-page.jsx

```diff
@@ -63,6 +72,45 @@ const AdvertiserPage = () => {
                   name: nickname,
               });
     };
+    const { data: p2p_advert_info, isSuccess: has_p2p_advert_info } = useP2PAdvertInfo(counterparty_advert_id, {
+        enabled: !!counterparty_advert_id,
+    });
+
+    const setShowAdvertInfo = React.useCallback(
+        () => {
+            if (has_p2p_advert_info && is_advertiser && !is_barred) {
+                const { is_active, is_visible } = p2p_advert_info || {};
+                const advert_type = p2p_advert_info?.type === buy_sell.BUY ? 1 : 0;
+
+                if (is_active && is_visible) {
+                    advertiser_page_store.setActiveIndex(advert_type);
```

Comment:
Could u please destructure the advertiser_page_store and buy_sell_store too :pray:

Reply:
as we have discussed, better to not destructure these now as with the ongoing refactor, a lot has already changed, and would cause further conflicts in this page,, so no need to destructure for now 🙏